frontend/assets/application/conoscenze_ambientali/css/skin-home.css

```diff
@@ -1,10 +1,10 @@
-.home {
+body {
     background:url("images/desk2.jpg") no-repeat scroll center top #f2f2f2;
 	/*background-size:100%;*/
 	}
 
 @media (max-width: 767px) {
-.home {
+body {
```

Comment:
In general, if you change a rule on body you can affect many things, so I want to be sure you checked before merging